pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -741,14 +751,42 @@ monitorLoop:
 			logger.Reason(err).Error("failed to get domain job info")
 			break
 		}
+		log.Log.Infof("Debug2: stats-type	%d", stats.Type)
+		log.Log.Infof("%d, %d, %d, %d, %d", libvirt.DOMAIN_JOB_UNBOUNDED, libvirt.DOMAIN_JOB_NONE, libvirt.DOMAIN_JOB_COMPLETED, libvirt.DOMAIN_JOB_CANCELLED, libvirt.DOMAIN_JOB_FAILED)
+
+		jobStatsInfo, err := dom.GetJobStats(0)
```

Comment:
I think you can just now replace `stats, err := dom.GetJobInfo()` on line 749 with `stats, err := dom.etJobStats(0)` instead of here

Reply:
Done